README.md

```diff
@@ -1,5 +1,7 @@
 Docker Compose
 ==============
+![Docker Compose](https://github.com/docker/compose/blob/master/docs/logo.png).
```

Comment:
I don't know if the .png needs to be in the same path for this to work, but it's worth trying